### PR TITLE
feat: test connect integration form

### PR DESCRIPTION
**Pull request by geisterfurz007**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

This PR adds a bit of component testing for the connection form (and a few tiny things while at it).

Tiny things being:
- An `any` that had no business not being a proper type
- A typo in `onCreateInteraction`

- **Why this change was needed?** (You can also link to an open issue here)

#1343 (+ tiny things because it doesn't hurt and cleaning up on the side is a good idea)

- **Other information**:

I had to change an import from `import styled from '@emotion/styled';` to `import styled from '@emotion/styled/macro';`. This is to aid with the new component tests failing with the error "Component selectors can only be used in conjunction with @emotion/babel-plugin". If there is a better way to deal with that (like updating some config somewhere that sets up babel for the cypress run or something along those lines), let me know! I searched around a bit and my understanding is that the babel config needs to reference `'@emotion'` in the `plugins` section (which is already the case in the .babelrc) but there is some trickery with monorepo setups. In a quick search I could only find emotion being used in the web project so I am unsure what else to do or try here 😕

### CODE REVIEW

File: apps/web/src/pages/integrations/components/ConnectIntegrationForm.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { useEffect, useState, useReducer } from 'react';
 import { Controller, useForm } from 'react-hook-form';
-import styled from '@emotion/styled';
+import styled from '@emotion/styled/macro';
```
Comment (BiswaViraj) [inline apps/web/src/pages/integrations/components/ConnectIntegrationForm.tsx:3]: Could you please explain why `macro` is needed?

**Review by scopsy — APPROVED** [review]
Thank you so much for the contribution. We can leave the macro import, I don't belive it will hurt of keeping it :) 

Do you mind just merging the conflict so we can merge this ?

**Comment by scopsy** [comment]
NVM, was an easy one. Just resolved it

**Comment by geisterfurz007** [comment]
Awesome, thanks! GitHub should have a notification for when your PR has conflicts, I missed that entirely, sorry!

**Pull request merged**